std/uni.d

```diff
@@ -2318,7 +2318,7 @@ public:
      * $(LI $(B %x) formats the intervals as a [low..high$(RPAREN) range of lowercase hex characters)
      * $(LI $(B %X) formats the intervals as a [low..high$(RPAREN) range of uppercase hex characters)
      */
-    void toString(scope void delegate(const(char)[]) sink,
+    void toString(Writer)(scope Writer sink,
```

Comment:
Shouldn't there be some template constraints for this? At the very least `is(typeof(put(sink, "")))`